lmnet/executor/profile_model.py

```diff
@@ -147,21 +175,46 @@ def helper(node, level):
         is_quant_kernel = all([layer not in node.name for layer in unquant_layers]) and "kernel" == \
                           node.name.split("/")[-1]
         bits = bit if is_quant_kernel else 32
+        node_name = "total" if level == 0 else node.name
         res.append(
-            [level, node.name, node.total_parameters, node.total_parameters * 32, node.total_parameters * bits])
+            [level, node_name, node.total_parameters, node.total_parameters * 32, node.total_parameters * bits])
         idx = len(res) - 1
+
+        node_param_dict = {
+            'name': node_name,
+            'parameters': node.total_parameters,
+            'size': node.total_parameters * 32,
+            'quant_size': node.total_parameters * bits,
+            'children': [],
+        }
+        if node_name == "total":
+            node_param_dict = {
+                'total_parameters': node.total_parameters,
+                'total_size': node.total_parameters * 32,
+                'quant_bit': bit,
+                'total_quant_size': None,
+                'children': [],
+            }
+
         sumsq = 0
         for c in node.children:
-            size_quat = helper(c, level + 1)
+            size_quat, children = helper(c, level + 1)
             sumsq += size_quat
+            node_param_dict["children"].append(children)
         res[idx][-1] = sumsq or res[idx][-1]
-        return res[idx][-1]
-
-    helper(prof, 0)
+        if node_name == "total":
+            node_param_dict["total_quant_size"] = res[idx][-1]
```

Comment:
`res[idx][-1]` appear several times.
It's better to add name as normal variable.

Reply:
@iizukak 
Thanks! Fixed 👍